src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user: { email, name } }) => (
```

Comment:
better to desctructure inside the function body, will look more readable this way